icu4c/source/i18n/ethpccal.h

```diff
@@ -330,6 +330,18 @@ class EthiopicAmeteAlemCalendar : public EthiopicCalendar {
      * @internal
      */
     virtual int32_t defaultCenturyStartYear() const override;
+
+    /**
+     * Compute the era from extended year.
+     * @internal
+     */
+    int32_t extendedYearToEra(int32_t extendedYear) const override;
+
+    /**
+     * Compute the year from extended year.
+     * @internal
+     */
+    int32_t extendedYearToYear(int32_t extendedYear) const override;
```

Comment:
It looks like you have the same thing here twice.  Is that intentional?

Reply:
There are two calnedar class here EthiopicCalendar and EthiopicAmeteAlemCalendar
each one of them have two functions extendedYearToEra and extendedYearToYear so there are total 4 added into this header

Reply:
Oh, wait.  It's two different function names.  Sorry, I missed that when I read this the first time.